python/paddle/static/amp/__init__.py

```diff
@@ -12,10 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ...fluid.contrib import mixed_precision
-from ...fluid.contrib.mixed_precision import *
-from ...fluid.contrib.mixed_precision import bf16
-from ...fluid.contrib.mixed_precision.bf16 import *
+from ...fluid.contrib.mixed_precision import decorate  # noqa: F401
+from ...fluid.contrib.mixed_precision import CustomOpLists  # noqa: F401
+from ...fluid.contrib.mixed_precision import AutoMixedPrecisionLists  # noqa: F401
+from ...fluid.contrib.mixed_precision import fp16_guard  # noqa: F401
+from ...fluid.contrib.mixed_precision import cast_model_to_fp16  # noqa: F401
+from ...fluid.contrib.mixed_precision import cast_parameters_to_fp16  # noqa: F401
+from ...fluid.contrib.mixed_precision import AutoMixedPrecisionListsBF16  # noqa: F401
+from ...fluid.contrib.mixed_precision import bf16_guard  # noqa: F401
+from ...fluid.contrib.mixed_precision import rewrite_program_bf16  # noqa: F401
+from ...fluid.contrib.mixed_precision import convert_float_to_uint16  # noqa: F401
 
-__all__ = mixed_precision.__all__
-__all__ += bf16.__all__
+from ...fluid.contrib.mixed_precision.bf16 import AutoMixedPrecisionListsBF16  # noqa: F401
+from ...fluid.contrib.mixed_precision.bf16 import bf16_guard  # noqa: F401
+from ...fluid.contrib.mixed_precision.bf16 import rewrite_program_bf16  # noqa: F401
+from ...fluid.contrib.mixed_precision.bf16 import convert_float_to_uint16  # noqa: F401
```

Comment:
These codes are duplicated and should be removed.

Reply:
done